python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -52,10 +52,13 @@ class ShotgunAPI(object):
     COMMAND_SUCCEEDED = 0
     COMMAND_FAILED = 1
 
-    BASE_CONFIG_URI = os.environ.get(
-        "TK_BOOTSTRAP_CONFIG_OVERRIDE",
-        "sgtk:descriptor:app_store?name=tk-config-basic"
-    )
+    OVERRIDE_CONFIG_PATH = os.environ.get("TK_BOOTSTRAP_CONFIG_OVERRIDE")
```

Comment:
Do we need to duplicate this logic here? why is it needed? is it to get a cache key? Could we instead add better interfaces to the toolkit manager so we can retrieve the bootstrap config uri?